rust/datafusion/src/datasource/csv.rs

```diff
@@ -36,6 +36,7 @@ pub struct CsvFile {
 }
 
 impl CsvFile {
+    /// Initializer
```

Comment:
I'd question if this rustdoc comment adds any value since `new` is the standard name for an initializer?

Reply:
I added an `#[allow(missing_docs)]` for each of these, paired with a TODO on the struct itself to add an example of usage.